README.md

```diff
@@ -7,17 +7,293 @@ SDK for the Seam API written in Ruby.
 
 ## Description
 
-TODO
+[Seam](https://seam.co) makes it easy to integrate IoT devices with your applications.
+This is an official SDK for the Seam API.
+Please refer to the official [Seam Docs](https://docs.seam.co/latest/) to get started.
+
+Parts of this SDK are generated from always up-to-date type information
+provided by [@seamapi/types](https://github.com/seamapi/types/).
+This ensures all API methods, request shapes, and response shapes are
+accurate and fully typed.
 
 ## Installation
 
-Add this as a dependency to your project using [Bundler] with
+Add this as a dependency to your project using [Bundler](https://bundler.io/) with:
 
-```
+```bash
```

Comment:
Minor point, but now this is the only repo that does this. I think I may have omitted it before because this is not actually bash due to the prompt `$`
```suggestion
```
```

Reply:
10d27a358ea01bf7c1d3d2524c6e28d259a68234